src/SFA.DAS.Authorization.Mvc/ActionDescriptorExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿#if NET462
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web.Mvc;
+
+namespace SFA.DAS.Authorization.Mvc
+{
+    public static class ActionDescriptorExtensions
+    {
+        private static readonly ConcurrentDictionary<string, List<DasAuthorizeAttribute>> Cache = new ConcurrentDictionary<string, List<DasAuthorizeAttribute>>();
+
+        public static IEnumerable<DasAuthorizeAttribute> GetDasAuthorizeAttributes(this ActionDescriptor actionDescriptor)
```

Comment:
If we 'ToList()' the 'Concat()' result, then we should probably follow the maxim 'return the most specific type and accept the most generic type' and return a List (or possibly IList).
However, there doesn't seem a need to convert it to a list at this point, so perhaps we should remove the 'ToList()'.